cpp/src/barretenberg/plonk/composer/standard_composer.cpp

```diff
@@ -64,7 +62,7 @@ std::shared_ptr<plonk::proving_key> StandardPlonkComposerHelper::compute_proving
     // Initialize circuit_proving_key
     // TODO(#392)(Kesha): replace composer types.
     circuit_proving_key = proof_system::initialize_proving_key<Flavor>(
```

Comment:
Probably should just extract this CircuitType from flavor, or just move responsibility of encoding this information in vk to a responsibility outside of Bb.